spec/datacaster_spec.rb

```diff
@@ -57,32 +57,32 @@
   end
 
   describe "default typecasting" do
-    subject { described_class.schema { default(5) } }
+    subject { described_class.schema { default([{}]) } }
```

Comment:
Leave old specs intact. Add new single "it" spec (which would not use this "subject"), wherein test that attempt to change the returned default value would lead to exception